### PR TITLE
feat: test case for RHMI cleanup

### PR DESCRIPTION
**Pull request by psturc**
# Description
New test case for RHMI failed uninstallation alerts + cleanup
https://issues.redhat.com/browse/INTLY-8272

### CODE REVIEW

**Comment by psturc** [comment]
@trepel fyi

**Comment by aidenkeating** [comment]
:eyes:

File: test-cases/tests/uninstallation/o01-verify-cloud-resources-can-be-properly-cleaned-up-in-case-of.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+```
+
+5. Go to alert manager
+
+```bash
+open "https://$(oc get routes alertmanager-route -n redhat-rhmi-middleware-monitoring-operator -o jsonpath='{.spec.host}')"
+```
+
+> Verify that PostgresResourceDeletionStatusPhaseFailed and RedisResourceDeletionStatusPhaseFailed alerts are firing
+> (It should take around 5 minutes for these alerts to start firing. If they are not firing yet, wait for couple of minutes and refresh the browser page)
```
Comment (darahayes) [inline test-cases/tests/uninstallation/o01-verify-cloud-resources-can-be-properly-cleaned-up-in-case-of.md:61]: minor nit: you could mention that you should see the alerts go into a pending state pretty much right away.

**Pull request merged**